kubert/src/shutdown.rs

```diff
@@ -47,49 +56,40 @@ pin_project_lite::pin_project! {
 
 /// Creates a shutdown channel
 ///
-/// [`Shutdown`] watches for `SIGINT` and `SIGTERM` signals. When a signal is received, [`Watch`]
-/// instances are notifed and, when all watches are dropped, the shutdown is completed. If a second
-/// signal is received while waiting for watches to be dropped, the shutdown is aborted.
+/// [`Shutdown`] watches for `SIGINT` and `SIGTERM` signals on Linux or Ctrl-C and Ctrl-Break on
+/// Windows. When a signal is received, [`Watch`] instances are notifed and, when all watches are
+/// dropped, the shutdown is completed. If a second signal is received while waiting for watches
+/// to be dropped, the shutdown is aborted.
 ///
 /// If a second signal is received while waiting for shutdown to complete, the process
 #[cfg_attr(docsrs, doc(cfg(feature = "shutdown")))]
 pub fn sigint_or_sigterm() -> Result<(Shutdown, Watch), RegisterError> {
```

Comment:
I don't think that this API makes sense on Windows. That this is called 'sigint_or_sigterm' is explicitly about handling things on Linux. We should probably make this function unix-specific.